Src/DependencyCollector/Net45.Tests/DependencyTrackingTelemetryModuleTest.cs

```diff
@@ -63,7 +63,7 @@ public void TestHttpPostRequestsAreCollected()
                 var item = (DependencyTelemetry)sentTelemetry;
                 Assert.AreEqual(url, item.Data);
                 Assert.AreEqual(url.Host, item.Target);
-                Assert.AreEqual("POST " + url.AbsolutePath, item.Name);
```

Comment:
rolling back change done by @brahmnes in https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/449.

This in NET45 tests and NET45 remains instrumented with EventSource or Profiler only